python/node_classification.py

```diff
@@ -731,7 +733,12 @@ def predict(
 
     predicted_class = int(pred.detach().cpu().numpy()[position])
 
-    return mgp.Record(predicted_class=predicted_class, status="Prediction done.")
+    return mgp.Record(
+        predicted_class=current_values[HeteroParams.INV_LABEL_REINDEXING][
+            predicted_class
+        ],
+        status="Prediction done.",
```

Comment:
Sounds better :slightly_smiling_face: 
```suggestion
        status="Prediction complete.",
```